Increase flexibility in creating meshes and loading meshes (#266)
adamantine-sim/adamantine: README.md

    * mesh\_format: abaqus, assimp, unv, ucd, dbmesh, gmsh, tecplot, xda, vtk,
    vtu, exodus, or default, i.e., use the file suffix to try to determine the
    mesh format (required)
    * mesh\_scale\_factor: Apply a uniform scaling factor to the mesh (e.g. if the mesh is defined in mm or inches instead of m), (default value: 1)

>     * set\_material\_id\_to\_0: Clear the material IDs defined in the mesh and sets them all to zero: true or false (default value: false)

Reviewer: I think I would call this something like `set_material_id` or `set_uniform_material_id`. Use don't need to know/don't care which value we are using.

Author: How about `reset_material_id`, and then mention that we set it to zero in the description? I think the value being zero is important because it corresponds to the `material_0` material input block.

```suggestion
    * reset\_material\_id: Clear the material IDs defined in the mesh and set them all to zero so all material properties are given by the `material\_0` input block: true or false (default value: false)
```

Reviewer: Sounds good.

  * if import\_mesh is false:
    * length: the length of the domain in meters (required)
    * height: the height of the domain in meters (required)
    * width: the width of the domain in meters (only in 3D)

>     * length_origin: the reference location in the length direction (default value: 0)

Reviewer: ```suggestion
    * length\_origin: the reference location in the length direction (default value: 0)
```

>     * height_origin: the reference location in the height direction (default value: 0)

Reviewer: ```suggestion
    * height\_origin: the reference location in the height direction (default value: 0)
```

>     * width_origin: the reference location in the width direction (only in 3D) (default value: 0)

Reviewer: ```suggestion
    * width\_origin: the reference location in the width direction (only in 3D) (default value: 0)
```
